server/devdocs/selectors/function-comments.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Find the node containing the docblock
+ * for a function given its name
+ *
+ * JSDoc function comments are attached to AST nodes
+ * in a JavaScript file but these aren't always with
+ * the direct nodes defining the functions.
+ *
+ * For example, if a function is exported such as
+ * below we end up with this comment being associated
+ * with the export declaration and the function itself
+ * is a child of the export declaration node.
+ *
+ * Therefore we need to find the "outermost" node onto
+ * which an associated docblock could be attached for
+ * a function of the given name.
+ *
+ * @see http://eslint.org/parser/ or similar tool for an interactive AST explorer
```

Comment:
[AST explorer](https://astexplorer.net/) is also a great option which supports Espree and a number of other parsers.